kuksa_databroker/Dockerfile

```
@@ -15,44 +15,27 @@
 
 # This builds KUKSA databroker
 
-FROM rust:1.65-alpine as base
-RUN apk add git musl-dev python3 protoc ncurses-terminfo-base make 
-RUN rustup component add rustfmt
+FROM messense/rust-musl-cross:x86_64-musl AS builder-amd64
```

```suggestion
FROM ghcr.io/rust-cross/rust-musl-cross:x86_64-musl AS builder-amd64
```

since we're running on github, it would presumably be better to use the GHCR registry ([rust-musl-cross prebuild images](https://github.com/rust-cross/rust-musl-cross#prebuilt-images))